deps/libcaption/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(CAPTION_HEADERS
 )
 
 add_library(caption STATIC ${CAPTION_SOURCES})
+install_obs_headers("caption/caption.h")
```

Comment:
does this take care of headers pulled by caption.h ?

For me it pulled in more headers afterwards.

Reply:
Doh, I did assume this was the only public header, I dont have a setup to compile an out of project plugin atm. Will update with all the headers. Thanks.

Reply:
Looks like obs-internal was unintentionally leaked anyway. That should prevent the need for any of these deps headers.

Reply:
well if obs-internal will no longer be needed and with that the caption headers are no longer needed. I am fine too.